lib/brew.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+import os
+from shutil import which
+
+from devenv.lib import fs
+from devenv.lib import proc
+
+repo_path = "/opt/homebrew"
```

Comment:
Can you rename this to something like "brew_install_path". I started to think we were installing brew into the sentry repo